src/arweave/composite-client.ts

```diff
@@ -642,14 +643,25 @@ export class ArweaveCompositeClient
     };
   }
 
-  async getData({ id }: { id: string }): Promise<ContiguousData> {
+  async getData({
+    id,
+    region,
+  }: {
+    id: string;
+    region?: Region;
```

Comment:
what's the reasoning for using `region` instead of `range` here?

Reply:
We're using `region` in other places like tx-chunk. I just kept the "convention"

Reply:
I used `Region` instead of `Range` because, in my mind, a range should specify `start` and `end`, and we're specifying `offset` and `size`.

Reply:
I was surprised to see this was needed too. My intent with the multi-range request ticket was that we implement this (passing regions down) there and use that as part of the multi-range handling. The multi-range handling we implemented does seem relatively efficient though, so I'm not too concerned atm.